lib/morse_translator.rb

```diff
@@ -38,6 +41,18 @@ def initialize
                     "9" => "----.",
                     "0" => "-----",
                     " " => " "}
+    # @translate = translate
+  end
+  def translate
```

Comment:
This probably gave you a `wrong number of arguments` error when you ran your test. Be sure to follow those error messages! You needed to add an argument here so that your method looked something like the following:

```
def translate(message)
  
end
```

That `message` in parenthesis after `translate` is an *argument*. It allows you to pass values into the method.